### ISSUE TITLE
Change parent for a bunch of promoter proximal promoters

### ISSUE THREAD

**Issue by ValWood** [The-Sequence-Ontology/SO-Ontologies#442]
And refine def
(these are implicitly promoter proximal because the def already mentions promoter, this is just more precise)

Ace2_UAS
Update def: An RNA polymerase II *proximal* promoter element

iron_repressed_GATA_element
Update def: An RNA polymerase II *proximal* promoter element

CRE rename CRE_Proximal
Update def: An *RNA polymerase II proximal* promoter element

CSL_response_element
Update def: An *RNA polymerase II proximal* promoter element

MCB
Update def: An *RNA polymerase II proximal* promoter element

CuRE
Update def: An *RNA polymerase II proximal* promoter element

DRE
Update def: An *RNA polymerase II proximal* promoter element

PCB
Update def: An *RNA polymerase II proximal* promoter element

FLEX_element
Update def: An *RNA polymerase II proximal* promoter element

GATA_box
Update def: An *RNA polymerase II proximal* promoter element

HSE
Update def: An *RNA polymerase II proximal* promoter element

AP_1_binding_site
Update def: An *RNA polymerase II proximal* promoter element

CDRE_motif
Update def: An RNA polymerase II *proximal* promoter element

STREP_motif
Update def: An *RNA polymerase II proximal* promoter element

forkhead_motif
Update def: An RNA polymerase II proximal promoter element

zinc_repressed_element
Update def: An RNA polymerase II proximal promoter element

CCAAT_motif
Update def: An RNA polymerase II proximal promoter element

TR_box
Update def: An RNA polymerase II proximal promoter element

**Comment by ValWood** [comment]
Change parent to
http://www.sequenceontology.org/browser/current_svn/term/SO:0001668

**Comment by ValWood** [comment]
Outstanding, I don't know if these are always proximal and some are  not restricted to yeast

AACCCT_box
A conserved 17-bp sequence (5'-ATCA(C/A)AACCCTAACCCT-3') commonly present upstream of the start site of histone transcription units functioning as a transcription factor binding site.

sterol_regulatory_element

(For the others, if they  are used outside yeast the papers refer to a promoter proximal context)

@pgaudet
@RLovering

**Comment by ValWood** [comment]
+ mating_type_M_box

**Comment by ValWood** [comment]
@RLovering
@ukemi

based on the current defs do you agree with this change?

**Comment by RLovering** [comment]
Hi Val

thanks for pointing to this ticket and also for your comments on the non-mammalian motifs/elements as there are obviously a lot to cover. 

To be honest my concern is that I am not sure that some of these are 'always' in the proximal promoter. The other concern is that GATA_box  is currently a child of core_promoter_element (no definition) whereas CSL_response_element  is a child of promoter_element (no definition but synonym is general transcription factor binding site, core promoter element which is worrying). Neither of these terms have a relation to promoter. possibly it would be safer to just put them under regulatory_element. 

Basically I could look up every motif/element in pubmed and see where it appears to lie wrt promoter/enhancer etc, but I would rather get people who are working in this area to confirm which ones are always proximal to the promoter (and have the definition for this region agreed) and which ones are always in enhancers and which ones can be in either region, if indeed these distinctions can be made.

Ruth

**Comment by ValWood** [comment]
Good points.
None of the terms we requested are "core promoter", and references to "core promoter' should be removed.
I'm not sure which ones are present outside fungi. Fungal specific ones  could be "promoter proximal", but as you say the safest option would be to move all to 

TF_binding_site 
http://www.sequenceontology.org/miso/current_svn/term/SO:0000235

I think the reason they were placed under promoter element is that this is broad in SO. It has 2  descendants
regulatory_promoter_element
core_promoter_element

but it is also undefined so this isn't very clear.....

I see what you mean ;) I wonder why there are no definitions...

**Comment by davidwsant** [comment]
Hi @ValWood  and @RLovering,

The structure of the terms relating to regulatory regions has been restructured. The terms with no definitions have now also been updated to have definitions. Some of the terms in mention have moved, such as iron_repressed_GATA_element is now a child of GATA_box(SO:0001840), which is a child of core_eukaryotic_promoter_element(SO:0001660), which is a child of promoter_element(SO:0001659). 

It looks like you were requesting that many terms that are mostly under [promoter_element SO:0001659](http://sequenceontology.org/browser/current_svn/term/SO:0001659) be moved to [TF_binding_side SO:0000235](http://sequenceontology.org/browser/current_svn/term/SO:0000235) . Promoter_element overlaps TF_binding_site. Considering that terms have been restructured, would you mind looking over the changes and determine if you would still like to make any changes? If you would like to move terms, please give specifics.

Thank you,

Dave Sant

**Comment by ValWood** [comment]
Hi David.
I can see some issues. I will open tickets and tag the relevant people. It might not be today...

v

**Comment by davidwsant** [comment]
Hi Val,

Thanks for looking at them. I think opening new tickets is a good idea. I will close out this issue and look for the new ones as they come.

Thanks,

Dave
